<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Students</class>
 <widget class="QWidget" name="Students">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Студенти</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonCreateList">
       <property name="text">
        <string>Створити список</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOpenList">
       <property name="text">
        <string>Відкрити існуючий</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEditSurname">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Прізвище</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditName">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Ім'я</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAdd">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Додати</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDelete">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Видалити</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButtonUpdate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Оновити</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPrint">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Надрукувати список</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
